layers/+lang/prolog/README.org

```diff
@@ -0,0 +1,92 @@
+#+TITLE: prolog layer
+
+# TOC links should be GitHub style anchors.
+* Table of Contents                                        :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+  - [[#leader][Leader]]
+    - [[#consulting][Consulting]]
+    - [[#compiling][Compiling]]
+    - [[#formatting][Formatting]]
+    - [[#inserting][Inserting]]
+    - [[#help][Help]]
+    - [[#evaluating][Evaluating]]
+
+* Description
+This layer adds support for Prolog using the bundled Prolog mode for Emacs. In addition it also adds ediprolog support for better interaction with SWI-Prolog. 
+
+** Features:
+  - Designed for SWI-Prolog as a default, but can be used with other Prologs that Prolog mode supports. 
+  - Interactive consulting and compiling.
+  - Auto-formatting.
+  - Apropos and help lookup.
+
+    
+* Install
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =prolog= to the existing =dotspacemacs-configuration-layers= list in this
+file.
+
+Note that the layer currently uses the Prolog mode that is bundled with Emacs. 
+If you want to use the separately maintained [[https://bruda.ca/_media/emacs/prolog.el][Prolog Mode by Stefan D. Bruda]]:
+- Place =prolog.el= in the =local/prolog= directory relative the layer directory.
+- When defining the =prolog-packages= for the layer, change =(prolog :location built-in)=
+ to =(prolog :location local)=.
+
+* Key bindings
+  
+** Leader
```

Comment:
Please remove the `Leader` headline and move current third level headlines to the second level please.